fcrepo-http-api/src/test/java/org/fcrepo/http/api/FedoraLdpTest.java

```diff
@@ -692,7 +691,7 @@ public void testGetWithExternalMessageBinary() throws Exception {
 
 
     @Test
-    @SuppressWarnings({"resource", "unchecked"})
```

Comment:
There is now a warning about a potential resource leak

Reply:
Fixed